README.md

```diff
@@ -14,6 +14,8 @@
 - [Getting started](#getting-started)
   - [Installation](#installation)
   - [Code sample](#code-sample)
+    - [Standard upload](#standard-upload)
```

Comment:
@szekelyzol could you confirm it should be `Regular upload` as in this doc: https://docs.api.video/vod/upload-a-video-regular-upload?

Reply:
@ThibaultBee the original phrase was "regular" in the docs.